adjust_restart_for_new_land_cover/adjust_restart_for_new_land_cover.py

```diff
@@ -71,7 +78,7 @@ def prepare_mapping(nVeg, ConfigFile):
     
     return MappingConf
 
-def setup_output_dataset(OutputVegetation, VariableList):
+def setup_output_dataset(OutputVegetation, InputDataset, VariableList):
```

Comment:
```suggestion
def setup_output_dataset(OutputVegetation):
```
I think these arguments aren't being used anymore

Reply:
Yep you're right. Removed.